README.md

```diff
@@ -27,6 +28,11 @@ Notes for myself, but feel free to contribute if you have things to share :D
 - LiveMemTracer [tested : green] : https://github.com/cesarl/LiveMemTracer
 - IGMemTracer [not tested] : https://github.com/deplinenoise/ig-memtrace
 
+#### OpenGL tools
+
+- ApiTrace : https://github.com/apitrace/apitrace
```

Comment:
I think that Apitrace and remotery should be place in `Profiling` section no ? Next to microprofile.